packages/store/src/gateway/cgwClient.ts

```diff
@@ -11,6 +11,12 @@ export const getBaseUrl = () => {
 }
 export const rawBaseQuery = fetchBaseQuery({
   baseUrl: '/',
+  credentials: 'include',
+  headers: {
+    'Content-Type': 'application/json',
+    Accept: 'application/json',
+    'Set-Cookie': 'HttpOnly;Secure;SameSite=None',
```

Comment:
This has no effect as it is a response header. This needs to be set by the service when setting e.g. an auth cookie.